### PR TITLE
Refacto prerequisites

### PR DESCRIPTION
**Pull request by jnduwamu**
The files debian.yml and centos.yml in the task directory were merged into os_prerequisites.yml file to prevent repetitive code
The task that install unzip is modified to be installed whatever the os by using the following module : 
`- name: Install unzip
  package:
    name: unzip
    state: present`

Since the name of the java package is different on debian os and on centos we'll install it accordingly on the two systems by using the condition `when`

### CODE REVIEW

File: tasks/os_prerequisites.yaml [hunk 1]
```diff
@@ -44,7 +67,7 @@
     dest: "{{ install_path }}"
     owner: "{{ service_user |d('root') }}"
     group: "{{ service_user |d('root') }}"
-    mode: '0644'
+    mode: '0755'
```
Comment (NicoGGG) [inline tasks/os_prerequisites.yaml:70]:
```suggestion
```

File: tasks/os_prerequisites.yaml [hunk 2]
```diff
@@ -1,16 +1,38 @@
...
+  lineinfile:
+    path: /etc/environment
+    regexp: 'JAVA_HOME'
+    line: "JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-amd64"
+  tags: ['java', 'env']
+  when: ansible_facts['os_family'] == "Debian"
+
 
 - name: Create Install Path Dir
   file:
```
Comment (NicoGGG) [inline tasks/os_prerequisites.yaml:38]: Remove this task. This is done in install_prep.yml

**Review by NicoGGG — CHANGES_REQUESTED** [review]

File: tasks/os_prerequisites.yaml (hunk 1 above)
Comment (jnduwamu) [reply tasks/os_prerequisites.yaml]: Done

**Comment by jnduwamu** [comment]
Done

**Comment by jnduwamu** [comment]
Conflict handled

**Review by NicoGGG — APPROVED** [review]

**Pull request merged**